src/ontology/cl-edit.owl

```diff
@@ -6644,9 +6650,12 @@ AnnotationAssertion(owl:deprecated obo:CL_0000381 "true"^^xsd:boolean)
 
 # Class: obo:CL_0000382 (scolopale cell)
 
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.1017/CBO9780511818202.024") Annotation(oboInOwl:hasDbXref "doi:https://doi.org/10.1016/B978-012369493-5.50012-2") obo:IAO_0000115 obo:CL_0000382 "A cell that surrounds the dendrite of a scolopidial neuron.")
```

Comment:
> Consider adding "...and that is part of a scolopodium.".

In my mind this was implicit by the “surrounds a scolopidial neuron”, but no objection to adding it explicitly.

> Consider describing the function:
>> Integumental movement causes the scolopale to lengthen, compressing the dendrite and triggering a receptor potential by affecting the opening of ion channels.

That is way too specific for CL. The mechanosensory mechanism of scolopidia may vary even across the scolopidia of a single species (for example, not all scolopidia in _Drosophila melanogaster_ may be sensitive to stretching along the long axis – some are sensitive to lateral vibrations instead), so I can’t imagine the variability across insects, and even less across arthropods. As I mentioned in another PR, in CL I’d rather sometimes say nothing rather than risking saying too much.